spec/factories/user.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    email { "testuser@test.com" }
+    password { "123456" }
+    role { 0 }
```

Comment:
```suggestion
    role { :user }
```
I think you can use this syntax in case of enum, but I am not sure. If you will check it please let me know.

Reply:
Yup you can! Looks much clearer now :D.